openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
+    post:
```

Comment:
There is POST for aksks, but could not see a DELETE fo aksks

Reply:
Check line no. 149

Reply:
nothing wrong with the naming convention . 
A resource can be a singleton or a collection. For example, “customers” is a collection resource and “customer” is a singleton resource (in a banking domain). We can identify “customers” collection resource using the URI “/customers”. We can identify a single “customer” resource using the URI “/customers/{customerId}”.
https://restfulapi.net/resource-naming/

Reply:
likewise , here "/aksks" is used , which is a collection of aksk